extension/popup/popupController.jsx

```diff
@@ -169,6 +172,15 @@ export const PopupController = function() {
     });
     window.close();
   };
+  const initSettings = async () => {
+    const result = await settings.getSettingsAndOptions();
+    setUserSettings(result.settings);
+    setUserOptions(result.options);
+  };
```

Comment:
We don't want to add more initialization here, especially since it will almost never be needed - only if you use this one intent will this be necessary. If you look in `historyView.jsx:History()` you'll see an example of using an effect inside the component to fetch data. In the same way you could fetch this data inside the `MusicCard` component, and then it would only be fetched when the card was displayed.